js/lykke2.js

```diff
@@ -0,0 +1,1174 @@
+        }
+        const response = await this.publicGetOrderbooks (this.extend (request, params));
+        const payload = this.safeValue (response, 'payload', []);
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 assetPairId: 'BTCUSD',
+        //                 timestamp: '1643298038203',
+        //                 bids: [
+        //                     {
+        //                         "v":0.59034382,
+        //                         "p":36665.329
+        //                     }
+        //                 ],
+        //                 asks: [
+        //                     {
+        //                         "v":-0.003,
+        //                         "p":36729.686
+        //                     }
+        //                 ]
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const orderbook = this.safeValue (payload, 0, {});
+        const timestamp = this.safeString (orderbook, 'timestamp');
+        return this.parseOrderBook (orderbook, symbol, timestamp, 'bids', 'asks', 'p', 'v');
+    }
+
+    parseTrade (trade, market) {
+        //
+        //  public fetchTrades
+        //
+        //     {
+        //         "id":"71df1f0c-be4e-4d45-b809-c108fad5f2a8",
+        //         "assetPairId":"BTCUSD",
+        //         "timestamp":1643345958414,
+        //         "volume":0.00010996,
+        //         "price":37205.723,
+        //         "side":"buy"
+        //      }
+        //
+        //  private fetchMyTrades
+        //         {
+        //             "id":"813a3ffa-1c4b-45cb-b13f-1c077ea2748b",
+        //             "timestamp":1644155923357,
+        //             "assetPairId":"BCHEUR",
+        //             "orderId":"1b367978-7e4f-454b-b870-64040d484443",
+        //             "role":"Taker",
+        //             "side":"sell",
+        //             "price":280.569,
+        //             "baseVolume":0.01,
+        //             "quoteVolume":2.8056,
+        //             "baseAssetId":"2a34d6a6-5839-40e5-836f-c1178fa09b89",
+        //             "quoteAssetId":"EUR",
+        //             "fee":null
+        //         }
+        //
+        const marketId = this.safeString (trade, 'assetPairId');
+        const symbol = this.safeSymbol (marketId, market);
+        const id = this.safeString2 (trade, 'id', 'id');
+        const orderId = this.safeString (trade, 'orderId');
+        const timestamp = this.safeInteger (trade, 'timestamp');
+        const price = this.safeString2 (trade, 'price', 'price');
+        let amount = this.safeString2 (trade, 'volume', 'amount');
+        if (amount === undefined) {
+            amount = this.safeString2 (trade, 'baseVolume', 'amount');
+        }
+        const side = this.safeStringLower (trade, 'side');
+        const fee = {
+            'cost': 0, // There are no fees for trading.
```

Comment:
We must use parseNumber ('0') because we are going to switch to string-based representations with Precise soon: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings